### ISSUE TITLE
Alt Text to the CancelDlg's Icon

### ISSUE THREAD

**Issue by roaladje** [wixtoolset/issues#6218]
Please provide answers to the following questions to help us narrow down, reproduce, and fix the problem. Fill out one section and delete the others.

## Triage

Issues are triaged at [online meetings](https://www.firegiant.com/blog/), generally held alternate Tuesdays at 9:30 a.m. Pacific time (UTC-7/UTC-8). Meeting requests are sent to the [wix-devs](http://wixtoolset.org/documentation/mailinglist/#wix-devs) mailing list.

## Discussions and support

Note that discussions are best held on the mailing lists, not in issues. Please do not open issues requesting support or debugging help; first start a thread on the [wix-users](http://wixtoolset.org/documentation/mailinglist/#wix-users) mailing list and open an issue if the discussion suggests a bug in the WiX Toolset or that a feature request is appropriate.

## Bugs

If this issue is a bug:

* Which version of WiX are you building with? 

> 3.10

* Which version of Visual Studio are you building with (if any)?

> 2019.16.6.2

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> N/A

* Which version of .NET are you building with?

> 4.5

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Windows 10

* Describe the problem and the steps to reproduce it.

> 
I'm trying to make my app accessible (based on WCAG 2.1 guidelines). I discovered that the default CancelDlg of WiX Toolset, which is the confirmation dialog that pops when the user tries to cancel the installation, is not accessible.

There is a yellow warning icon beside the text, but it has no Alt text or Name. That makes narrators say "Image" instead of something along the line of "Warning icon" and confuse who is blind.

* Describe the behavior you expected and how it differed from the actual behavior.

> (Be as specific as you can and err on the side of providing too much information.)

## Feature requests

If this issue is a feature request:

* Describe the scenario and benefits that the feature supports.

> Help who is blind

* Describe how you're accomplishing the feature today (if possible).

> It may be possible to create a custom cancel dialog with an Alt Text (but feels wrong to do so)

* Describe what you'd like the new feature to do.

> Alt text in CancelDlg (WCAG 2.1 guidelines)
